modules/burials/lib/burials/pdf_fill/va21p530ez.rb

```diff
@@ -648,7 +651,14 @@ def convert_location_of_death
 
           @form_data.delete(location)
 
-          location_of_death['location'] = 'nursingHomeUnpaid' if location == 'atHome'
+          if location == 'atHome'
+            # This section is formatted in a wonky way to make the linter happy
+            location_of_death['location'] = 'nursingHomePaid' if home_hospice_care && home_hospice_care_after_discharge
+            if !home_hospice_care || !home_hospice_care_after_discharge
+              location_of_death['location'] =
+                'nursingHomeUnpaid'
+            end
```

Comment:
This is DeMorgan's law, so u should just be able to use else for the second conditional instead of `!home_hospice_care || !home_hospice_care_after_discharge`. Does the linter complain with this suggested change?
```suggestion
            if home_hospice_care && home_hospice_care_after_discharge
               location_of_death['location'] = 'nursingHomePaid'
            else
               location_of_death['location'] = 'nursingHomeUnpaid'
            end
```

Reply:
Well actually it auto-fixed it to this:
```ruby
location_of_death['location'] = if home_hospice_care && home_hospice_care_after_discharge
'nursingHomePaid' 
```

Reply:
gotcha, the linter be crazy sometimes.